docs/stats-engine/variance.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Variance 
+sidebar_label: Variance
+slug: /stats-engine/variance
+---
+
+The variance of the metric delta is required for computing its confidence interval and p-value.  It can be obtained directly from the variances of the test and control group means, which require different calculations depending on the type of metric.
```

Comment:
Should we call this section standard error?  This would differentiate it from the variance of the metric, and align with the terminology used in console.